clients/admin/src/components/GetSettings.tsx

```diff
@@ -0,0 +1,168 @@
+      ...updatedSettings[index],
+      value: {
+        ...updatedSettings[index].value!,
+        editable: true,
+      },
+    };
+    setSettings(updatedSettings);
+  };
+
+  const handleSave = async (index: number) => {
+    const setting = settings[index];
+    if (setting && setting.value) {
+      const updatedSettings = [...settings];
+      updatedSettings[index] = {
+        ...updatedSettings[index],
+        value: {
+          ...updatedSettings[index].value!,
+          editable: false,
+        },
+      };
+      setSettings(updatedSettings);
+
+      try {
+        const response = await fetch(`/api/setting/${setting.dataField}`, {
+          headers: {
+            Authorization: 'Basic ' + btoa('ykartal@ogu.edu.tr:sdfasdfadfasdfasdfasdf'),
+            'Content-Type': 'application/json',
+          },
+          method: 'PUT',
+          body: JSON.stringify(setting.value),
+        });
+
+        if (response.ok) {
+          console.log('Değer başarıyla kaydedildi.');
+        } else {
+          console.log('Değer kaydedilirken bir hata oluştu.');
+        }
+      } catch (error) {
+        console.log(error);
+      }
+    }
+  };
+
+  const handleInputChange = (index: number, newValue: string) => {
+    const updatedSettings = [...settings];
+    updatedSettings[index] = {
+      ...updatedSettings[index],
+      value: {
+        ...updatedSettings[index].value!,
+        value: newValue,
+      },
+    };
+    setSettings(updatedSettings);
+  };
+
+  return (
+    <div className="container mx-auto py-8">
+      <h1 className="text-3xl font-bold mb-4">Ayarlar</h1>
+      <div className="bg-white shadow overflow-hidden sm:rounded-md">
+        <table className="min-w-full divide-y divide-gray-200">
+          <thead>
+            <tr>
+              <th className="px-6 py-3 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider font-bold w-24">
+                Alan
+              </th>
+              <th className="px-6 py-3 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider font-bold w-24">
+                Değer
+              </th>
+              <th className="px-6 py-3 bg-gray-100"></th>
+            </tr>
+          </thead>
+          <tbody>
+            {settings.map((setting: SettingRow, index: number) => (
+              <tr key={index}>
+                <td className="px-6 py-4 w-48 font-bold">{setting.field}</td>
+                <td className="px-6 py-4">
+                  {setting.value && setting.value.editable ? (
+                    <div className="w-full border-gray-300 rounded-md p-2 focus:outline-none focus:border-blue-500">
+                      <textarea
+                        value={setting.value.value}
+                        onChange={(e) => handleInputChange(index, e.target.value)}
+                        className="w-full h-16 resize-none overflow-auto"
+                        style={{ maxHeight: '8rem' }}
```

Comment:
tablodaki kaymaları kontol etmek için değer sütunu içeriklerine şöyle bir styling yapılabilir:
style={{ width: '100%', resize: 'vertical', maxWidth: '800px', minWidth: '400px', wordWrap: 'break-word'}}